src/UI/Buyer/src/app/shared/services/favorites/favorites.service.spec.ts

```diff
@@ -4,8 +4,9 @@ import { OcMeService, MeUser } from '@ordercloud/angular-sdk';
 import { of, BehaviorSubject } from 'rxjs';
 import { FavoriteProductsService } from '@app-buyer/shared/services/favorites/favorites.service';
 import { AppStateService } from '@app-buyer/shared/services/app-state/app-state.service';
+import { ToastrService } from 'ngx-toastr';
 
-describe('FavoriteProductsService', () => {
+fdescribe('FavoriteProductsService', () => {
```

Comment:
The `f` off describe will probably want to be taken off.

Reply:
Good call :)